lib/src/main/java/com/scalar/admin/kubernetes/Pauser.java

```diff
@@ -79,26 +80,50 @@ public PausedDuration pause(int pauseDuration, @Nullable Long maxPauseWaitTime)
 
     TargetSnapshot target;
     try {
-      target = targetSelector.select();
+      target = getTarget();
```

Comment:
I separated this `targetSelector.select()` as one method `getTarget()` for testing. This targetSelector is initialized in the constructor of `Pauser`. So, controlling the behavior of `targetSelector` by using Mockito is a bit complex and difficult. That's why I separate this function.